src/Microsoft.FeatureManagement.AspNetCore/InlineDisabledFeatureHandler.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.FeatureManagement.Mvc
     /// <summary>
     /// A disabled feature handler that wraps an inline handler.
     /// </summary>
-    class InlineDisabledFeaturesHandler : IDisabledFeaturesHandler
+    sealed class InlineDisabledFeaturesHandler : IDisabledFeaturesHandler
```

Comment:
Breaking change.

Reply:
I recalled someone saying sealing classes is a breaking change, but the warning message convinced me it was okay:

[CA1852](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1852)	Type 'InlineDisabledFeaturesHandler' can be sealed because it has no subtypes in its containing assembly and is not externally visible

If we think it's still breaking- or just don't want to risk it I can disable this rule.

Reply:
Oh it's not a public type. Then indeed it can be sealed without a breaking change.